lib/flutrans.dart

```diff
@@ -104,20 +102,28 @@ class MidtransTransaction {
   final List<MidtransItem> items;
   final bool skipCustomer;
   final String customField1;
+  final String customField2;
+  final String customField3;
+
   MidtransTransaction(
     this.total,
     this.customer,
     this.items, {
-    this.customField1,
     this.skipCustomer = false,
+    this.customField1,
+    this.customField2,
+    this.customField3,
   });
+
   Map<String, dynamic> toJson() {
     return {
       "total": total,
-      "skip_customer": skipCustomer,
+      "customer": customer,
```

Comment:
Customer is an object, this will not work

Reply:
thank you!